php/class-ad-layers-post-type.php

```diff
@@ -106,7 +106,7 @@ public function manage_custom_columns( $column, $post_id ) {
 					$value = get_post_meta( $post_id, $column, true );
 					if ( ! empty( $value ) ) {
 						if ( is_array( $value ) ) {
-							$value = implode( ', ', $value );
+							$value = implode( ', ', array_map( function ( $value ) { return $value['ad_unit']; }, $value ) );
```

Comment:
This block is shared by four different columns, so really you should split this condition out just for `ad_layer_ad_units`.

Also, this would be better handled using the `wp_list_pluck` function.